### DISCUSSION TITLE
安卓app如何购买会员？

### DISCUSSION THREAD

**Discussion by robinspt** [gitbobobo/StreamMusic#339] [Q&A] [answered]
请问安卓app如何购买会员？通过支付宝购买是灰色锁定状态，无法点击。
<img src="https://github.com/gitbobobo/StreamMusic/assets/46096863/c604d092-9e7f-4813-a641-a52021cd85d6" alt="描述文字" width="300">

**Comment by gitbobobo** [comment, marked as answer]
杀掉重进可以购买，建议多用几次再做决定

**Comment by digua-cn** [comment]
主题 有些什么风格能截图看看吗
